src/pages/product/components/product-buttons-group/index.tsx

```diff
@@ -28,22 +28,18 @@ const ProductButtonsGroup: React.FC<ProductButtonsGroupProperties> = ({
 
 	return (
 		<StyledButtonsContainer>
-			<BaseButton
+			<StyledBaseButton
 				startIcon={<productIcons.ShopIcon />}
+				sx={{ pointerEvents: "none" }}
 				variant="primary_black_bold"
 			>
 				{t("ProductPage.addToCart")}
-			</BaseButton>
-			{!isPreviewMode && (
-				<BaseButton variant="secondary_black">
-					{t("ProductPage.makeAnOffer")}
-				</BaseButton>
-			)}
+			</StyledBaseButton>
 			<Box display="flex" gap="32px">
 				<BaseButton
 					onClick={handleLikeButtonClick}
 					startIcon={isLikeClicked ? <FavoriteIcon /> : <FavoriteBorderIcon />}
-					sx={{ fill: "black", textTransform: "none" }}
+					sx={{ fill: "black", pointerEvents: "none", textTransform: "none" }}
```

Comment:
```suggestion
					sx={{ fill: theme.palette,primary.black pointerEvents: "none", textTransform: "none" }}
```
or add custom black and use it